install/kubernetes/cilium/templates/hubble-ui-ingress.yaml

```diff
@@ -22,7 +22,14 @@ spec:
         paths:
           - path: /
```

Comment:
I just did a quick test, its failed for me due to below issue.

```
The Ingress "hubble-ui" is invalid: spec.rules[0].http.paths[0].pathType: Required value: pathType must be specified
```

Further checking the docs, I found this [one](https://kubernetes.io/docs/concepts/services-networking/ingress/)

> Each path in an Ingress is required to have a corresponding path type. Paths that do not include an explicit pathType will fail validation. There are three supported path types:

Reply:
Indeed, not sure how it worked with me (probably the `--dry-run`) but now that should be fixed.